src/sockperf.cpp

```diff
@@ -3114,6 +3101,14 @@ static int set_sockets_from_feedfile(const char *feedfile_name) {
             continue;
         }
 
+        if (tmp->server_addr.ss_family == AF_UNIX) {
+            s_user_params.tcp_nodelay = false;
+            strncpy(s_user_params.addr.addr_un.sun_path, tmp->server_addr.addr_un.sun_path, MAX_UDS_NAME);
+            s_user_params.addr.ss_family = AF_UNIX;
+            tmp->server_addr = s_user_params.addr;
```

Comment:
You're copying in one direction and then in another. Since we're filling `s_user_params` here, it's enough to do one assignment:
```suggestion
            s_user_params.addr = tmp->server_addr;
```

Reply:
yes sorry of course